tests/21-build_docker.sh

```diff
@@ -60,5 +60,55 @@ stest 1 sudo singularity build -F "$CONTAINER" docker://something_that_doesnt_ex
 stest 1 singularity exec "$CONTAINER" true
 stest 1 singularity exec "$CONTAINER" false
 
+# make sure local test does not exist, ignore errors
+sudo docker kill registry >/dev/null 2>&1
+sudo docker rm registry >/dev/null 2>&1
+
```

Comment:
I'm not sure about testing, but if there are multiple processes running on the same server we should be wearing of some kind of race condition here.

Reply:
right, you cannot have two processes running the tests at the same time, and dare you not have another docker container in your system called "registry"! ;)
How do you propose to solve this, would it be enough to rename "registry" to, say,  "singularity_test_registry", or just let docker pick a unique name for us?

Reply:
I'm not actually sure if there would be a race condition - it would depend on how the CI is running it (which I'm not sure of). I think we can likely leave as is (it's working!) and if something fishy comes up, look into a possible race. That's the one great and bad thing about CI - it's easy because it's a black box that just sets things up for you, and it's hard because it's a black box that sets things up for you :)